.ci/checker-framework.groovy

```diff
@@ -168,7 +168,7 @@ private static String getOsSpecificCmd(final String cmd) {
 private static List<CheckerFrameworkError> getErrorFromText(final List<List<String>> errorsList) {
     final List<CheckerFrameworkError> errors = new ArrayList<>()
     final Pattern errorExtractingPattern = Pattern
-        .compile('.*[\\\\/](checkstyle[\\\\/]src.*\\.java):\\[(\\d+)[^]]*][^\\[]*\\[([^]]*)](.*)')
+        .compile('.*[\\\\/](src.*\\.java):\\[(\\d+)[^]]*][^\\[]*\\[([^]]*)](.*)')
```

Comment:
`[\\\\/]src.*\\` -> `[\\\\/]src[\\\\/].*\\?`

Reply:
`[\\\\/]src.*\\` does not exist, note the capture group delimited by the `(` in `[\\\\/](src.*\\`

Reply:
Updated first comment. I was looking at the older version of the changes.

Reply:
don't know what a srcaptain is, but done :)

Reply:
Senior Captain .